MediaBrowser.Common/Plugins/LocalPlugin.cs

```diff
@@ -106,6 +106,11 @@ public override int GetHashCode()
         /// <inheritdoc />
         public bool Equals(LocalPlugin other)
         {
+            if (!(other is LocalPlugin))
```

Comment:
shouldn't it just be `other == null`? It's either null or LocalPlugin (or a subclass of LocalPlugin)

Reply:
That's what i did first time around. But @crobibero overrode == which calls Equals, so you end up in an endless loop.

Reply:
Use https://docs.microsoft.com/en-us/dotnet/api/system.object.referenceequals?view=net-5.0